### PR TITLE
Ui changes

### PR DESCRIPTION
**Pull request by 17kpatrick**
Made changes specified in the UI changes issue.

### CODE REVIEW

File: public/js/chat.js [hunk 1]
```diff
@@ -229,7 +229,7 @@ $(function () {
         $('#myModal').modal('hide')
         connected = true;
         // Display the welcome message
-        var message = "Welcome to Snake game Chat – ";
+        var message = "Rainbow game";
         log(message, {
```
Comment (agomez314) [inline public/js/chat.js]: We still want it to say, "Welcome to Rainbow Game Chat"

File: public/index.html [hunk 2]
```diff
@@ -46,17 +46,18 @@ <h2>scores go here</h2>
...
-                                       <span style="color: orange">w</span> game!</h1>
-                    <p>Enter a username to being playing</p>
+                    <h1>Welcome to the <br /><span style="color:yellow">R</span>
+                                       <span style="color:orange">A</span>
+                                       <span style="color:blue">I</span>
+                                       <span style="color:green">N</span>
+                                       <span style="color:red">B</span>
+                                       <span style="color:yellow">O</span>
+                                       <span style="color: orange">W</span> Game!</h1>
                     <form action="" style="padding-top:20px;">
```
Comment (agomez314) [inline public/index.html]: Capitalizing first letter is fine, instead of the whole word.

Comment (17kpatrick) [reply public/index.html]: According to the requirements the game is supposed to be RAINBOW Game with the first word in all caps. Should I still only capitalize the first letter?

Comment (agomez314) [reply public/index.html]: if that's the case, let's go with the requirements

Comment (17kpatrick) [reply public/index.html]: Alright, sounds good

**Comment by agomez314** [comment]
Good job on the turnaround 👍🏼

**Pull request merged**